### ISSUE TITLE
Пожелание

### ISSUE THREAD

**Issue by foshechka** [Chiaki/VKBK#1]
Чувствую что-то будет интересное. Сделай, пожалуйста, хороший плеер, как в ВК примерно и возможность поиска: фото, видео, аудио ...

**Comment by Chiaki** [comment]
Доброго времени суток.
Спасибо за отзыв. Фильтры планируются в будущем, сейчас основной приоритет это именно бэкап данных (диалоги, стена, документы). Если у вас есть на примете более удобные плееры с лицензией MIT или аналогичной, то я с удовольствием рассмотрю варианты.
По фото планирую сделать фильтр по размеру, может быть дате, хотя хронологически они идентичны вконтакту.

**Comment by foshechka** [comment]
Замечательно! Поддерживаю и желаю успеха в дальнейшем развитии!

**Comment by Phatalist** [comment]
Я правильно понимаю что на данный момент нет возможности сдампить переписку?

**Comment by Chiaki** [comment]
Да. Дамп комментариев ещё находится в to do. А для личной переписки, которая тоже была запланирована, ВК поменял API в августе 2017. Так что сейчас нет возможности получать "исходящие" сообщения.

**Comment by Phatalist** [comment]
ну ладно :) В любом случае огромное спасибо за уникальный в своем роде инструмент!

**Comment by Chiaki** [comment]
@Phatalist хорошая новость: дампить переписку уже можно. ^_-
Пока в альфа версии (0.8.0), могут быть баги, но сам функционал уже есть.

**Comment by junkym0nk3y** [comment]
Я тоже пару пожеланий выскажу, если никто не против )

Можно ли сделать так, что бы закачка изображений/файлов не обрывалась, если не удалось получить одно из изображений? А то весит 300 файлов и первый в очереди не может скачать, а потом обрывается автоматическая закачка всех последующих. 
Удалить из очереди его можно только через бд, получается. Такая фигня.

Было бы круто, если бы ещё и "избранные" скачивались, хотя бы изображения. Для меня это последний бастеон перед удалением своего профиля ВК. 

Заранее благодарен.

**Comment by Chiaki** [comment]
Проблема с "дохлыми" ссылками у ВКонтакте есть и у меня. Хотя помнится они уверяли что никогда ничего не удаляется. В ближайшей версии планирую добавить решение для этой проблемы - можно будет скрывать такие пункты в очереди закачки.

А что за "избранные"? Расскажите подробнее, возможно стоит синхронизировать и их.

**Comment by junkym0nk3y** [comment]
https://vk.com/fave - в общем говоря. 
Туда попадает всё, что ты лайкаешь. Посты/Изображения/Видео

**Comment by Chiaki** [comment]
Возможно в будущем добавится. Хотя обычно лучше сохранять к себе в альбом, дабы не удалилось ничего "случайно".

**Comment by phatal** [comment]
@Chiaki Спасибо за добавление дампа переписки)) сделать бэкап и забыть о вк как никогда актуально :)

**Comment by ecub** [comment]
@Chiaki Привет и спасибо за то, что делаешь такую крутую штуку! Возможно из-за незнакомых еще типов аттачей или моей тупости при синхронизации переписки дампер иногда уходит в вечную загрузку. Запили, пожалуйста, возможность пропускать часть таких сообщений, или диалог полностью, или просто игнорирование подобного рода ошибок:

<img width="1408" alt="2018-09-06 16 49 36" src="https://user-images.githubusercontent.com/9007645/45161603-f11ac480-b1f4-11e8-9523-f3dca06d7855.png">

**Comment by ecub** [comment]
Кстати, а что насчет конференций, ты думал о том, чтобы также парсить их?

**Comment by Chiaki** [comment]
@ecub пожалуйста!
Могу ошибиться, но эта ошибка должна уже быть исправлена. Попробуйте перекачать свежие файлы и обновить (кроме конфига естественно (: ). Если ошибка сохранится, то оформите issue, чтобы можно было разобраться в чём дело и не засорять данную ветку.
Так как если есть незнакомый тип чего-либо, то он просто игнорируется.

Конференции вроде и так должны подтягиваться. Но данный раздел ещё довольно сырой, так что баги неизбежны. Проверю.
